### PR TITLE
update syntax wxpython

### PR DESCRIPTION
**Pull request by davideziviani**
wxpython with py3.6+

### CODE REVIEW

**Comment by davideziviani** [comment]
@ibell : can you look into this PR? I have a few more things to push but I would like to merge this one first.

**Comment by ibell** [comment]
I checked out the PR, seems to be more or less working.  What about the splash screen?

File: GUI/panels/scroll_panels.py [hunk 1]
```diff
@@ -846,10 +846,14 @@ def get(key):
...
-                    core += chunk
-                    
+            try:
+                for key,chunk in plugin_chunks.iteritems():
+                    if key in allowed:
+                        core += chunk
+            except:
+                for key,chunk in plugin_chunks.items():
+                    if key in allowed:
+                        core += chunk
```
Comment (ibell) [inline GUI/panels/scroll_panels.py:856]: Only the second one is needed I believe

Comment (davideziviani) [reply GUI/panels/scroll_panels.py:856]: Splash screen works fine; issues are on the output panels as the results are not visible for some (possibly encoding) issues

Comment (davideziviani) [reply GUI/panels/scroll_panels.py:856]: image

**Comment by ibell** [comment]
On my side, splash screen doesn't work, but maybe that is a linux issue

**Comment by davideziviani** [comment]
I am more concerned about the output panel. I can look into the Splash Screen. I am sure I pushed all the changes

**Comment by ibell** [comment]
Also, on my side, the state chooser is broken (only the first state is visible, pressure is hidden): 

image

**Comment by ibell** [comment]
Should we merge this one and then try to figure out Output Panel?

**Comment by davideziviani** [comment]
image

**Comment by davideziviani** [comment]
works for me. I am still running py3.6.x, but should not be a problem

**Comment by ibell** [comment]
So don't worry about state chooser then...

**Comment by davideziviani** [comment]
Let's merge this one and work on OutputPanel and other issues. The GUI works fine for me on Win.

**Comment by davideziviani** [comment]
My wxpython version is 4.0.4; I can update to 4.0.7 to see if there are other issues.

**Comment by ibell** [comment]
Parametric table also broken: 

```
Traceback (most recent call last):
  File "/home/ian/Code/pdsim/GUI/panels/pdsim_panels.py", line 1493, in OnAddTerm
    option = ParametricOption(self, GUI_objects)
  File "/home/ian/Code/pdsim/GUI/panels/pdsim_panels.py", line 1176, in __init__
    labels = [o.annotation for o in GUI_objects.itervalues()]
AttributeError: 'dict' object has no attribute 'itervalues'
```

**Comment by ibell** [comment]
k, I'll merge now then, and we can keep working on remaining issues.

**Pull request merged**